components/example/Status.vue

```diff
@@ -39,6 +39,9 @@ defineProps({
           </tr>
         </tbody>
       </table>
+      <div class="mt-2 text-gray-400">
+        Data is fetched from the server every two seconds.
```

Comment:
let's make `two` a variable here that uses the interval setting, so that we don't forget to update this texty in the future!